src/polyfills/array-includes.js

```diff
@@ -44,6 +44,9 @@ function includes(value, opt_fromIndex) {
 */
 export function install(win) {
   if (!win.Array.prototype.includes) {
-    win.Array.prototype.includes = includes;
+    win.Object.defineProperty(Array.prototype, 'includes', {
+        enumerable: false,
```

Comment:
huh, you are right, native `includes` is `writable` and `configurable`. Did not expect that...